client/components/AllPrompts.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { getAllPromptsThunk } from '../store'
+import PropTypes from 'prop-types'
+
+import { Link } from 'react-router-dom'
+import Card from '@material-ui/core/Card'
+import CardContent from '@material-ui/core/CardContent'
+
+class AllPrompts extends Component {
+  constructor() {
+    super()
+
+    this.state = {}
+  }
+
+  componentDidMount() {
+    this.props.getAllPrompts()
+  }
+
+  render() {
+    const { prompts } = this.props
+    const promptKeys = Object.keys(prompts)
+
+    return (
+      <div>
+        <h1>All Prompts</h1>
+
+        {promptKeys &&
+          promptKeys.map((promptId) => (
+            <Card key={promptId}>
+              <CardContent>
+                <Link to={`/admin/single-prompt/${promptId}`}>
```

Comment:
it would be good to add the routes to routes.js in the same PR if you are setting them up here

Reply:
you can have the route go to something like `Home`